docs/CHANGELOG.md

```diff
@@ -18,7 +20,7 @@ Changed the return types of
 
 **Closed issues:**
 
-eisop#376, eisop#532.
+eisop#376, eisop#494, eisop#532.
```

Comment:
#494 is this PR, not an issue. I think we don't need this entry, there already is an entry above.

Reply:
Sure, thanks